ogr/ogr_geometry.h

```diff
@@ -2193,9 +2193,17 @@ class CPL_DLL OGRCompoundCurve : public OGRCurve
     virtual void
     assignSpatialReference(const OGRSpatialReference *poSR) override;
 
-    OGRErr addCurve(const OGRCurve *, double dfToleranceEps = 1e-14);
-    OGRErr addCurveDirectly(OGRCurve *, double dfToleranceEps = 1e-14);
-    OGRErr addCurve(std::unique_ptr<OGRCurve>, double dfToleranceEps = 1e-14);
+    /** Default relative tolerance to assume that the end of the previous curve
+     * is equal to the start of the next one.
+     */
+    static constexpr double DEFAULT_TOLERANCE_EPS = 1e-14;
```

Comment:
```suggestion
    static constexpr double DEFAULT_TOLERANCE_EPSILON = 1e-14;
```
just for readability

Reply:
done